### PR TITLE
release 1.0.0 final changes [geonode 4.1.2]

### LINKED ISSUE #72: Bug: Celery container is not starting properly and exiting right away
## Bug Description

when deploying geonode-k8s the celery container continously crashes, e.g. using minikube deployment

## Additional Information

Somehow env var: `IS_CELERY` is not set anymore in current main branch

### PR DESCRIPTION
**Pull request by mwallschlaeger**
## Description

This PR fixes minor issues with the chart and prepares for the new chart release 1.0.0. 
* fixes issue related to missing IS_CELERY var in celery deployment (#72) 
* increases Celery memory due to experienced OOMKilled (#73)
* update docs (#52)
* pycsw microservice is disabled by default, as its missing some changes to geonode which will be released with version 4.2 (https://github.com/GeoNode/geonode/pull/11319)
* now using 52north/geonode:4.1.2 image, as long as official images are not ready (https://github.com/GeoNode/geonode-project/issues/471), geonode-k8s will get a new minor release as soon as the new images are there (#66)

## Type of Change

Please select the relevant option:

- [x] Bug fix
- [ ] New feature
- [x] Documentation update
- [ ] Refactoring
- [ ] Other (please describe)

## Related Issue

If there is an existing issue related to this pull request, please reference it here.

closes #72 #73 #66 #52

## Checklist

Please ensure that your pull request meets the following requirements:

- The pull request is limited to one type (docs, feature, bug fix, etc.)
- The pull request is as small as possible. Consider opening multiple pull requests instead of one large one.
- The feature or bug fix has been discussed and documented in an issue beforehand.

## Additional Notes

Any additional information or context regarding the pull request can be provided here.

Thank you for creating this pull request

### CODE REVIEW

File: charts/geonode/README.md [hunk 1]
```diff
@@ -193,9 +193,24 @@ Helm Chart for Geonode a web-based application and platform for developing geosp
...
...OperatesOn\": \"operateson\",\n      \"pycsw:OperatesOnIdentifier\": \"operatesonidentifier\",\n      \"pycsw:OperatesOnName\": \"operatesoname\",\n      \"pycsw:Degree\": \"degree\",\n      \"pycsw:AccessConstraints\": \"restriction_code\",\n      \"pycsw:OtherConstraints\": \"raw_constraints_other\",\n      \"pycsw:Classification\": \"classification\",\n      \"pycsw:ConditionApplyingToAccessAndUse\": \"conditionapplyingtoaccessanduse\",\n      \"pycsw:Lineage\": \"lineage\",\n      \"pycsw:ResponsiblePartyRole\": \"responsiblepartyrole\",\n      \"pycsw:SpecificationTitle\": \"specificationtitle\",\n      \"pycsw:SpecificationDate\": \"specificationdate\",\n      \"pycsw:SpecificationDateType\": \"specificationdatetype\",\n      \"pycsw:Creator\": \"creator\",\n      \"pycsw:Publisher\": \"publisher\",\n      \"pycsw:Contributor\": \"contributor\",\n      \"pycsw:Relation\": \"relation\",\n      \"pycsw:Links\": \"download_links\",\n  },\n}","pod_name":"pysw","port":8000,"replicaCount":1,"resources":{"limits":{"cpu":"500m","memory":"1Gi"},"requests":{"cpu":"500m","memory":"1Gi"}}}` | pycsw integration is based on https://github.com/geopython/pycsw/blob/master/docker/kubernetes |
```
Comment (ridoo) [inline charts/geonode/README.md:196]: I would not bloat the README so much. In doubt, just [remove documentation](https://github.com/zalf-rdm/geonode-k8s/blob/4c688f745491cbb8db1a154a5bb2a1c85be18ebf/charts/geonode/values.yaml#L371) for the whole `pycsw` part.

File: charts/geonode/README.md [hunk 2]
```diff
@@ -193,9 +193,24 @@ Helm Chart for Geonode a web-based application and platform for developing geosp
...
...eTypeVersion\": \"servicetypeversion\",\n      \"pycsw:Operation\": \"operation\",\n      \"pycsw:CouplingType\": \"couplingtype\",\n      \"pycsw:OperatesOn\": \"operateson\",\n      \"pycsw:OperatesOnIdentifier\": \"operatesonidentifier\",\n      \"pycsw:OperatesOnName\": \"operatesoname\",\n      \"pycsw:Degree\": \"degree\",\n      \"pycsw:AccessConstraints\": \"restriction_code\",\n      \"pycsw:OtherConstraints\": \"raw_constraints_other\",\n      \"pycsw:Classification\": \"classification\",\n      \"pycsw:ConditionApplyingToAccessAndUse\": \"conditionapplyingtoaccessanduse\",\n      \"pycsw:Lineage\": \"lineage\",\n      \"pycsw:ResponsiblePartyRole\": \"responsiblepartyrole\",\n      \"pycsw:SpecificationTitle\": \"specificationtitle\",\n      \"pycsw:SpecificationDate\": \"specificationdate\",\n      \"pycsw:SpecificationDateType\": \"specificationdatetype\",\n      \"pycsw:Creator\": \"creator\",\n      \"pycsw:Publisher\": \"publisher\",\n      \"pycsw:Contributor\": \"contributor\",\n      \"pycsw:Relation\": \"relation\",\n      \"pycsw:Links\": \"download_links\",\n  },\n}"` | pycsw config file parameters, see docs: https://docs.pycsw.org/_/downloads/en/latest/pdf/ |
```
Comment (ridoo) [inline charts/geonode/README.md:202]: As above: Use `# -- @ignore` or (perhaps better) [`# -- @default <something>`](https://github.com/zalf-rdm/geonode-k8s/blob/4c688f745491cbb8db1a154a5bb2a1c85be18ebf/charts/geonode/values.yaml#L401) flag  here

**Review by ridoo — CHANGES_REQUESTED** [review]
What about the secret separation? Do you plan to include this in another release?

**Comment by mwallschlaeger** [comment]
> What about the secret separation? Do you plan to include this in another release?

no i want to wait until the secret separation is merged and add it to the release.

**Review by ridoo — APPROVED** [review]
Looks good

**Pull request merged**